addons/web/static/src/views/basic_relational_model.js

```diff
@@ -1034,12 +1034,15 @@ export class StaticList extends DataPoint {
         });
 
         const parentID = this.model.__bm__.localData[this.__bm_handle__].parentID;
-        this.model.__bm__.notifyChanges(parentID, {
+        await this.model.__bm__.notifyChanges(parentID, {
             order_line: {
                 operation: "MULTI",
                 commands: commandsWithId,
             },
         });
+
+        this.model.root.__syncData();
+        this.model.notify();
```

Comment:
c'est le js qui a proposé de modifier tout ça? :eyes:

Reply:
yep, I added the function following their instructions,  the last two lines `__syncData` and `notify` were recently added and are needed to apply changes to the rest of the lines and re-render the changes